Backend/src/main/java/com/pg/programmerground/service/PlaygroundService.java

```diff
@@ -24,29 +25,23 @@
 @RequiredArgsConstructor
 public class PlaygroundService {
```

Comment:
```suggestion
@Transactional(readOnly = true)
public class PlaygroundService {
```
서비스 클래스 위에 트랜잭션 어노테이션으로 readOnly 켜놓으면 하위 메소드들은 기본적으로 readOnly로 동작함
readOnly가 아닌 메소드의 경우 해당 메소드에 @Transaction만 붙여주면 됨

Reply:
참고로 CQRS 모델 또는 Read, Write 디비를 분리해서 구성하는 경우 XXXQueryService, XXXCommandService 이렇게 나눠지는데 위에 예시처럼 전역으로 트랜잭션 어노테이션을 붙이면 편함.